microprofile-fault-tolerance/src/main/java/org/jboss/eap/qe/microprofile/fault/tolerance/deployments/v10/HelloFallback.java

```diff
@@ -0,0 +1,23 @@
+package org.jboss.eap.qe.microprofile.fault.tolerance.deployments.v10;
+
+import javax.inject.Inject;
+
+import org.eclipse.microprofile.faulttolerance.ExecutionContext;
+import org.eclipse.microprofile.faulttolerance.FallbackHandler;
+
+/**
+ * Class providing @Fallback medhod for {@link HelloService} service
+ */
+public class HelloFallback implements FallbackHandler<String> {
+    @Inject
+    private MyContext context;
+
+    @Override
+    public String handle(ExecutionContext context) {
+        // WFLY-12982
```

Comment:
Having an issue ID in a comment might seem adequate to someone who knows what this is about. Can you please expand the comment a little bit and describe why it is mentioned here?

Reply:
ok, done

fixed